src/GitReleaseNotes/GitReleaseNotesEnvironment.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GitReleaseNotes
+{
+    public static class GitReleaseNotesEnvironment
+    {
+        #region Fields
```

Comment:
btw, just install this: https://visualstudiogallery.msdn.microsoft.com/0ca60d35-1e02-43b7-bf59-ac7deb9afbca